java/src/main/java/test/SpeakerModel.java

```diff
@@ -0,0 +1,17 @@
+package test;
+
+import com.sun.jna.PointerType;
+
+public class SpeakerModel extends PointerType implements AutoCloseable {
+	public SpeakerModel() {

```

Comment:
Indentation issue

Reply:
And no need for empty constructor actually

Reply:
The empty constructor is needed.  Not sure why exactly, but I get a runtime error if it's not there.

What is the indentation issue?

Reply:
You have wrong number of spaces

Reply:
If you have a particular coding standard, you need to let me know what it is. There is no standard for Java indentation. These are just sample programs